<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>janela_jogodavelha</class>
 <widget class="QDialog" name="janela_jogodavelha">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>385</width>
    <height>440</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>385</width>
    <height>440</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>385</width>
    <height>440</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Tic Tac Toe</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../img/keycap-number-sign_23-fe0f-20e3.png</normaloff>../img/keycap-number-sign_23-fe0f-20e3.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: white;</string>
  </property>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>7</width>
     <height>228</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 3px;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>7</width>
     <height>228</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: rgb(255, 255, 255);
border-radius: 3px;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>200</y>
     <width>229</width>
     <height>7</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 3px;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>280</y>
     <width>229</width>
     <height>7</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 3px;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>130</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: transparent;
color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: transparent;
color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>130</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: transparent;
color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn6">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: transparent;
color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn5">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: transparent;
color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn4">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>210</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: transparent;
color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn7">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: transparent;
color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn8">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>290</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: transparent;
color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="btn9">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>290</y>
     <width>68</width>
     <height>68</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>48</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: none;
background-color: transparent;
color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_turn">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>380</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>16</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_playagain">
   <property name="geometry">
    <rect>
     <x>15</x>
     <y>24</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #ffffff;
color: rgb(255, 255, 255);
background-color: rgb(40, 42, 58);
border-radius: 7px;</string>
   </property>
   <property name="text">
    <string>Play Again</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>90</y>
     <width>385</width>
     <height>361</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.506, y1:1, x2:0.494, y2:0, stop:0 rgba(235, 55, 90, 255), stop:1 rgba(240, 45, 115, 255));
border-radius: 10px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_turn_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>20</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Fredoka One</family>
     <pointsize>22</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(40, 42, 58);</string>
   </property>
   <property name="text">
    <string>TicTacToe</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <zorder>label</zorder>
  <zorder>line_4</zorder>
  <zorder>line_3</zorder>
  <zorder>line</zorder>
  <zorder>line_2</zorder>
  <zorder>btn1</zorder>
  <zorder>btn2</zorder>
  <zorder>btn3</zorder>
  <zorder>btn6</zorder>
  <zorder>btn5</zorder>
  <zorder>btn4</zorder>
  <zorder>btn7</zorder>
  <zorder>btn8</zorder>
  <zorder>btn9</zorder>
  <zorder>lbl_turn</zorder>
  <zorder>btn_playagain</zorder>
  <zorder>lbl_turn_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
